docs/upgrade/UPGRADE-v7.0.0-beta3.md

```diff
@@ -226,7 +228,7 @@ There you can find links to upgrade notes for other versions too.
             'route_label' => $product->getMainVariant()->getName(),
         ]);
         ```
-- *(optional)* rename Database tests to Functional tests
+- rename Database tests to Functional tests
```

Comment:
It should be `low priority`, shouldn't?

It was probably removed along with phing modifications where functional tests are mentioned. 
This exactly is, from my point of view, non-vital instruction and we made it vital just because of the phing target name.

Wouldn't be better to mention this rename along with instructions for phing targets near line 120 and keep this `low priority`?